frc_characterization/__main__.py

```diff
@@ -1,7 +1,7 @@
 # Shim to run the CLI entry point as an ordinary python module for those
 # who are having trouble adding python scripts to PATH.
 
-import cli.cli as cli
+import frc_characterization.cli.cli as cli
```

Comment:
The cli package has nothing else in it... Maybe we should just have it be a module?